http.c

```diff
@@ -2576,6 +2628,12 @@ int evhttp_connection_set_flags(struct evhttp_connection *evcon,
 	return 0;
 }
 
+void
+evhttp_connection_set_extended_method_cmp(struct evhttp_connection *evcon,
+	int (*cmp)(struct evhttp_extended_method *)) {
```

Comment:
I would prefer to see typedef for this

Reply:
I don't see typedef here, you mixed things a little bit, since you added typedef for `evhttp_set_cb` (while this patch is not about it), but skipped `evhttp_connection_set_extended_method_cmp`

Reply:
Oh, you just forgot convert this helper too (like `evhttp_set_extended_method_cmp`)

Reply:
let me check

Reply:
should be ok now

Reply:
Looks good, thanks!